tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
 
+invalid_file_list = (invalid_file, valid_with_ignore_rules_file,)
 ShellCheckBearTest = verify_local_bear(ShellCheckBear,
                                        valid_files=(valid_file,),
-                                       invalid_files=(invalid_file,))
+                                       invalid_files=invalid_file_list,
+                                       )
+
+ShellCheckBearTestIgnoreRules = verify_local_bear(
+    ShellCheckBear,
+    valid_files=(valid_with_ignore_rules_file,),
+    invalid_files=(),
+    settings={'shellcheck_ignore': ['SC2103', 'SC2164']},
+)
```

Comment:
Your indentation is not in sync with coala style here. See how `ShellCheckBearTest` is indented above. I suggest you rename `ShellCheckBearTestIgnoreRules` to something shorter, like `IgnoreRulesTest`.